Beta docs for compare changes (#5903)
fishtown-analytics/docs.getdbt.com: website/docs/docs/deploy/run-visibility.md

## Run history

> The **Run history** dashboard in dbt Cloud helps you monitor the health of your dbt project. It provides a detailed overview of all of your project's job runs and empowers you with a variety of filters to help you focus on specific aspects. You can also use it to review recent runs, find errored runs, and track the progress of runs in progress. You can access it from the top navigation menu by clicking **Deploy** and then **Run history**. 

Reviewer: ```suggestion
The **Run history** dashboard in dbt Cloud helps you monitor the health of your dbt project. It provides a detailed overview of all your project's job runs and empowers you with a variety of filters that enable you to focus on specific aspects. You can also use it to review recent runs, find errored runs, and track the progress of runs in progress. You can access it from the top navigation menu by clicking **Deploy** and then **Run history**. 
```

<Lightbox src="/img/docs/dbt-cloud/deployment/example-job-details.png" width="65%" title="Example of run details" />

### Run summary tab

> You can view or download in-progress and historical access logs for your dbt runs. This makes it easier for the team to debug errors more efficiently.

Author: ```suggestion
You can view or download in-progress and historical logs for your dbt runs. This makes it easier for the team to debug errors more efficiently.
```

<Lightbox src="/img/docs/dbt-cloud/deployment/access-logs.gif" width="85%" title="Access logs for run steps" />

### Lineage tab

> Displays the lineage graph associated with the job run so you can better understand the dependencies and relationships of the resources in your project. To view a node's metadata directly in [dbt Explorer](/docs/collaborate/explore-projects), select it (double-click) from the graph. 

Reviewer: ```suggestion
View the lineage graph associated with the job run so you can better understand the dependencies and relationships of the resources in your project. To view a node's metadata directly in [dbt Explorer](/docs/collaborate/explore-projects), select it (double-click) from the graph. 
```

<Lightbox src="/img/docs/collaborate/dbt-explorer/explorer-from-lineage.gif" width="85%" title="Example of accessing dbt Explorer from the Lineage tab" />

### Model timing tab <Lifecycle status="team,enterprise" /> 

> The **Model timing** tab displays the composition, order, and time taken by each model in a job run. The visualization appears for successful jobs and highlights the top 1% of model durations. This helps you identify bottlenecks in your runs, so you can investigate them and potentially make changes to improve their performance. 

Reviewer: ```suggestion
The **Model timing** tab displays the composition, order, and time each model takes in a job run. The visualization appears for successful jobs and highlights the top 1% of model durations. This helps you identify bottlenecks in your runs so you can investigate them and potentially make changes to improve their performance. 
```

You can find the dashboard on the [job's run details](#job-run-details). 

<Lightbox src="/img/docs/dbt-cloud/model-timing.png" width="85%" title="The Model timing tab displays the top 1% of model durations and visualizes model bottlenecks" />

### Artifacts tab

> Provides a list of the artifacts that were generated from the job run. dbt saves these files and you can download them if you want.

Reviewer: ```suggestion
This provides a list of the artifacts generated by the job run. The files are saved and available for download.
```

<Lightbox src="/img/docs/dbt-cloud/example-artifacts-tab.png" width="85%" title="Example of the Artifacts tab" />

### Compare tab <Lifecycle status="beta"/>

The **Compare** tab is shown for [CI job runs](/docs/deploy/ci-jobs) with the **Run compare changes** setting enabled. It displays details about [the changes from the comparison dbt performed](/docs/deploy/continuous-integration#compare-changes) between what's in your production environment and the pull request. To help you better visualize the differences, dbt Cloud highlights changes to your models in red (deletions) and green (inserts).

From the **Modified** section, you can view the following:

> - **Overview** &mdash; High-level summary about the changes to the models such as the number of primary keys that were added or removed. 

Reviewer: ```suggestion
- **Overview** &mdash; High-level summary about the changes to the models, such as the number of primary keys that were added or removed. 
```
